apps/builddao/widget/logout.jsx

```diff
@@ -11,7 +11,7 @@ const LogoutContainer = styled.div`
 
   img {
     width: 100%;
-    max-height: 100vh;
+    min-height: 100vh;
```

Comment:
multiple times... I'm imagining all these images that take up full view height; is this right?